test/e2e.integration.test.js

```diff
@@ -360,6 +360,24 @@ test('ledger : v2 contribution workflow with uphold BAT wallet', async t => {
   const surveyorIds = response.body.surveyorIds
   t.true(surveyorIds.length >= 5)
 
+ // look up surveyorIds to ensure that they belong to the correct cohorts
+  const ledgerDB = await connectToDb('ledger')
+  const surveyors = ledgerDB.collection('surveyors')
+
+  let numControlSurveryors = 0
+  let numGrantSurveyors = 0
+  for (let surveyorId of surveyorIds) {
+    let cohort = (await surveyors.findOne({surveyorId: surveyorId})).payload.cohort
```

Comment:
Nit: No need to keep it all on one line. Would appreciate doing just one step at a time.
1) do the query
2) access the cohort value